src/components/app_catalog/catalogs/index.js

```diff
@@ -30,24 +28,28 @@ class Catalogs extends React.Component {
                     <Link
                       className='app-catalog--repo'
                       key={this.props.catalogs.items[catalogName].metadata.name}
-                      to={'/apps/' + catalogName + '/'}
+                      to={'/app-catalogs/' + catalogName + '/'}
                     >
                       <div className='app-catalog--card'>
-                        <h3>
-                          {this.props.catalogs.items[catalogName].spec.title}
-                        </h3>
                         <img
                           src={
                             this.props.catalogs.items[catalogName].spec.logoURL
                           }
                         />
                       </div>
-                      <p>
-                        {
-                          this.props.catalogs.items[catalogName].spec
-                            .description
-                        }
-                      </p>
+                      <div className='app-catalog--description'>
+                        <h3>
+                          {this.props.catalogs.items[catalogName].spec.title}
+                        </h3>
+                        <p>
+                          <ReactMarkdown>
+                            {
+                              this.props.catalogs.items[catalogName].spec
+                                .description
+                            }
+                          </ReactMarkdown>
+                        </p>
```

Comment:
This is resulting in nested `<p>` tags, as ReactMardown is wrapping every paragraph in `<p></p>`, too. You can remove the outer one.